client/blocks/inline-help/popover.jsx

```diff
@@ -122,15 +219,16 @@ class InlineHelpPopover extends Component {
 			<div className={ classes }>
 				{
 					{
-						contact: <InlineHelpContactView />,
-						richresult: (
+						[ VIEW_CONTACT ]: <InlineHelpContactView />,
+						[ VIEW_RICH_RESULT ]: (
 							<InlineHelpRichResult
-								result={ this.props.selectedResult }
-								setDialogState={ this.props.setDialogState }
-								closePopover={ this.props.onClose }
+								result={ selectedResult }
+								setDialogState={ setDialogState }
+								closePopover={ onClose }
 							/>
 						),
-						checklist: <WpcomChecklist closePopover={ this.props.onClose } viewMode="prompt" />,
+						[ VIEW_CHECKLIST ]: <WpcomChecklist closePopover={ onClose } viewMode="prompt" />,
+						[ VIEW_ONBOARDING_WELCOME ]: <ChecklistOnboardingWelcome onClose={ onClose } />,
```

Comment:
_Explanation_: We may as well use the constant values as the object keys to reduce the chance of typos.